apps/taxes/src/modules/avatax/order-created/avatax-order-created-payload-transformer.ts

```diff
@@ -10,6 +10,14 @@ import { AvataxOrderCreatedPayloadLinesTransformer } from "./avatax-order-create
 export const SHIPPING_ITEM_CODE = "Shipping";
 
 export class AvataxOrderCreatedPayloadTransformer {
+  private matchDocumentType(config: AvataxConfig): DocumentType {
+    if (!config.isDocumentRecording) {
```

Comment:
I dont like the naming. `isDocumentRecording` sounds like "is this config some kind of document recording".

But in config context this is some part of it. So rather `documentRecordingEnabled` or `isDocumentRecordingEnabled`